src/server/bg_services/db_cleaner.js

```diff
@@ -58,23 +58,39 @@ async function clean_md_store(last_date_to_remove) {
         ${total_objects_count} objects - Skipping...`);
         return;
     }
+    const objects_to_remove = await clean_md_store_objects(last_date_to_remove, config.DB_CLEANER_DOCS_LIMIT);
```

Comment:
Did you rearrange the same code in smaller functions? I'm not sure if it's needed.

Reply:
Yes. looking to clean only deleted md objects and other calls in the function not needed. Divided the function into 3 sub functions.